### PR TITLE
[Artifactory] Readme update - oracle db configuration

### PR DESCRIPTION
**Pull request by rahulsadanandan**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -729,6 +729,20 @@ This can be done with the following parameters
 --set database.password=${DB_PASSWORD} \
 ...
 ```
+```bash
```
Comment (chukka) [inline stable/artifactory-ha/README.md:732]: @rahulsadanandan  can you add a heading - "Configuring Artifactory with external Oracle database" - Do same for artifactory chart

File: stable/artifactory-ha/README.md [hunk 2]
```diff
@@ -729,6 +729,20 @@ This can be done with the following parameters
...
+# To use artifactory with oracledb the required instant client library files, libaio has to be copied to tomcat lib. Also set LD_LIBRARY_PATH env variable. 
+...
+--set postgresql.enabled=false \
+--set artifactory.preStartCommand="mkdir -p /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib; cd /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib && wget -O instantclient-basic-linux.x64-19.6.0.0.0dbru.zip https://download.oracle.com/otn_software/linux/instantclient/19600/instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && unzip -jn instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && wget -O libaio1_0.3.110-3_amd64.deb http://ftp.br.debian.org/debian/pool/main/liba/libaio/libaio1_0.3.110-3_amd64.deb &&  dpkg-deb -x libaio1_0.3.110-3_amd64.deb . && cp lib/x86_64-linux-gnu/* ." \
```
Comment (chukka) [inline stable/artifactory-ha/README.md:736]: Also, Add a note if upgrading from 6.x to 7.x, they need add this under migration. preStartCommand

File: stable/artifactory-ha/README.md (hunk 1 above)
Comment (eldada) [inline stable/artifactory-ha/README.md:732]: Can you convert the long command line to a `values-oracle.yaml` snippet? I would rather users adopt the method of creating a custom `values-xyz.yaml` and passing it with a simple
```
helm upgrade --install artifactory-ha -f values-xyz.yaml jfrog/artifactory-ha
```
Cleaner and easier to digest...
 
Same for the `artifactory` README.

Comment (rahulsadanandan) [reply stable/artifactory-ha/README.md:752]: Corrected heading and added note .

Comment (rahulsadanandan) [reply stable/artifactory-ha/README.md:752]: Now its changed to yaml snippet.

File: stable/artifactory-ha/README.md [hunk 3]
```diff
@@ -730,6 +730,29 @@ This can be done with the following parameters
...
+  password: <DB_PASSWORD>
+artifactory:
+  preStartCommand: "mkdir -p /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib; cd /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib && wget -O instantclient-basic-linux.x64-19.6.0.0.0dbru.zip https://download.oracle.com/otn_software/linux/instantclient/19600/instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && unzip -jn instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && wget -O libaio1_0.3.110-3_amd64.deb http://ftp.br.debian.org/debian/pool/main/liba/libaio/libaio1_0.3.110-3_amd64.deb &&  dpkg-deb -x libaio1_0.3.110-3_amd64.deb . && cp lib/x86_64-linux-gnu/* ."  
+  extraEnvironmentVariables:
+  - name: LD_LIBRARY_PATH
+    value: /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib
+```
+2. Install Artifactory with the values file you created:
+```bash
+helm upgrade --install artifactory jfrog/artifactory --namespace artifactory -f values-oracle.yaml
```
Comment (eldada) [inline stable/artifactory-ha/README.md:753]: This should use `jfrog/artifactory-ha`.

File: stable/artifactory-ha/README.md [hunk 4]
```diff
@@ -730,6 +730,29 @@ This can be done with the following parameters
...
+```
+**NOTE:** If its an upgrade from 6x to 7x please add under migration. preStartCommand
```
Comment (eldada) [inline stable/artifactory-ha/README.md:755]: This should be
```
**NOTE:** If its an upgrade from 6.x to 7.x, add same `preStartCommand` under `artifactory.migration.preStartCommand`
```

File: stable/artifactory-ha/README.md (hunk 3 above)
Comment (rahulsadanandan) [reply stable/artifactory-ha/README.md]: corrected.

File: stable/artifactory-ha/README.md (hunk 4 above)
Comment (rahulsadanandan) [reply stable/artifactory-ha/README.md]: corrected now.

**Pull request merged**